.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
```

Comment:
unless this file can be kept in sync with CI (ie, changes to it will cause misformatted files to fail) i'd prefer the file not be included.